### ISSUE TITLE
LDCTimeComponent.py's LDCTimeType has a misspelled Enum name

### ISSUE THREAD

**Issue by rogerbock** [NextCenturyCorporation/AIDA-Interchange-Format#237]
UNKONWN should be UNKNOWN.

```
class LDCTimeType(Enum):
    ON = 1
    BEFORE = 2
    AFTER = 3
    UNKONWN = 4
```

This affects the TTL output because the string value of the Enum is used.

**Comment by craigwarsaw** [comment]
Thanks for reporting this issue.  We are planning on fixing this for an upcoming AIF v1.0.3 release

**Comment by craigwarsaw** [comment]
Resolved with the release of AIF v1.0.3
https://github.com/NextCenturyCorporation/AIDA-Interchange-Format/releases/tag/tag%2Fv1.0.3
